spec/spec_helper.cr

```diff
@@ -9,7 +9,6 @@ Spectator.configure do |config|
   config.randomize
 
   config.before_suite do
-    ENV.clear
```

Comment:
This clears `$PATH`, which is needed by `Process.run` to find the `coverage` script.